pinecone/sparse/bm25/bm25-quora.ipynb

```diff
@@ -0,0 +1,2451 @@
+{
```

Comment:
<p>We'll load the popular Quora dataset with precomputed embeddings. Both dense and sparse embeddings have been precomputed using the following models: </p><ul><li>Dense:&nbsp;<a href="https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2" target="_blank">sentence-transformers/all-MiniLM-L6-v2</a></li><li><br></li><li>Sparse: BM25</li></ul><blockquote><br></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>